### ISSUE TITLE
Multirepo discovery fails.

### ISSUE THREAD

**Issue by elrond1999** [kahole/vscode-magit#200]
Whenever I ask for magit status (alt-x g) and I have many repositories open in the workspace it will prompt for which repo to use.

It seems in your code it is supposed to discover this based on the uri of the file open in the editor. But that does not work. How can I debug this further?

Also it would be nice is "status.magit" buffer could be names with the repo is is showing status for.

**Comment by kahole** [comment]
You are correct it is supposed to try and figure out which repo it belongs to. Not sure why it doesnt in your case, but this has been somewhat a tricky problem in the past

**Comment by black7375** [comment]
~~This was the same when using monorepo.~~
~~Open a subproject of monorepo with VS Code and run `Magit status`, which will ask for the repository.~~

#253 setting resolved it.
Why not add it to the README?
